Patterns/Dutch/Dutch-Numbers.yaml

```diff
@@ -128,7 +130,7 @@ DoubleExponentialNotationRegex: !simpleRegex
 DoubleCaretExponentialNotationRegex: !simpleRegex
   def: (((?<!\d+\s*)-\s*)|((?<=\b)(?<!\d+,)))(\d+(,\d+)?)\^([+-]*[1-9]\d*)(?=\b)
 DoubleDecimalPointRegex: !paramsRegex
-  def: (((?<!\d+\s*)-\s*)|((?<=\b)(?<!\d+,)))\d+,\d+(?!(,\d+))(?={placeholder})
+  def: (?<=\b)((\d{1,3})(\.\d{3})*(\,\d+)?)(?={placeholder})
```

Comment:
Were these changes necessary? I don't recall if the previous form was trying to avoid some false positive situation...

Reply:
The original form causes the failure of the following test cases - 

**Number**
- "192.168.1.2"
- "291.890"

**Number with Unit**
- "In de gehele industrie daalde de olieproductie in dit land met 500.000 vaten per dag in de eerste acht maanden van dit jaar"
- "de kern van het bezit is 190.000 vierkante meter ongelooflijk duur vastgoed in de wijk Marunouchi, het zakelijke en financi\xeble centrum van Tokio, vaak gekscherend 'Mitsubishi-dorp' genoemd."
- "uit de gegevens van de handelsgroep over de gegevens van de handelsafdeling bleek dat de invoer in augustus, het op een na grootste maandelijkse totaal van het jaar, met 5% steeg ten opzichte van 1.458.000 ton in juli, maar onder het niveau van vorig jaar in juni 1988."
- "dus wanneer het psylliumgewas van volgend jaar in maart wordt geoogst, kan het kleiner zijn dan de 16.000 ton van de afgelopen paar jaar - precies op de top van de psylliumboom."
- "het project kost $46.8 miljoen, en is bedoeld om de productiecapaciteit van het bedrijf met 25% te verhogen tot 34.500 metrische ton koperkathodes per jaar."
- "de productie van canadese staal - ingots bedroeg 291.890 ton in de week eindigend op 7 oktober, 14. 8% meer dan het totaal van de vorige week, zei een Federaal agentschap."
- "Het gewicht van de zilverguldens tussen 1839 en 1954 heeft tot gevolg dat 1000 kg (= 1 ton) aan guldens een waarde heeft van 100.000 gulden. De gewichtseenheid krijgt daarmee een geldswaarde"